salt/client/ssh/__init__.py

```diff
@@ -896,9 +896,10 @@ def run_wfunc(self):
             opts_pkg['id'] = self.id
 
             retcode = 0
-
+            popts = copy.copy(self.context['master_opts'])
```

Comment:
Is there a reason this is a `copy()` and not a `deepcopy()`? It seems like the intention here is only to affect what's presented to the pillar, whereas this seems like it will modify the master opts for the context, which could be more than we want?